### PR TITLE
Ek develop tests 2017

### PR DESCRIPTION
**Pull request by ekaspern**
PRing these tests while I work on more tests.

### CODE REVIEW

File: src/components/info_pvr.coffee [hunk 1]
```diff
@@ -63,6 +63,7 @@ InfoPvr = React.createClass
       headerClass: ''
       pvrProps: {}
       className: 'plain-pvr'
+      close: ->
```
Comment (andyparisi) [inline src/components/info_pvr.coffee]: Looks like the prop `close` is set to `isRequired` in propTypes. We shouldn't follow this pattern because it essentially never fails. If the user doesn't pass a `close` method, then it will be covered by `getDefaultProps`.

Instead, we should remove this line, and let the component throw a propTypes error if the user doesn't pass it.

File: tests/components/info_pvr.coffee [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+    items = []
+
+    infoPvr = TestUtils.renderIntoDocument InfoPvr {
+      items: items
+    }
+
+    itemRowsNoInfo = TestUtils.findRenderedDOMComponentWithClass infoPvr, 'no-items'
+    itemRowsDiv = TestUtils.scryRenderedDOMComponentsWithClass infoPvr, 'plain-pvr-content-item'
+    
+    expect(itemRowsNoInfo.innerText).to.equal('No information to display')
```
Comment (andyparisi) [inline tests/components/info_pvr.coffee]: It looks like you've removed instances of `t()` inside the component...but shouldn't we _keep_ the `t()` function calls since you've defined `window.t` below?

If you remove the `t()` calls, you have to replace them with props. Here, the static text "No information to display" will be there, no matter which language the user requires.

After you put the `t()` calls back, you should also update this test to not check for this text directly, but just make sure the `no-items` node isn't null.

**Comment by ekaspern** [comment]
@andyparisi changes are done
